test/timeouts.go

```diff
@@ -22,7 +22,7 @@ import "time"
 
 const (
 	// Default timeout for tests in this package.
-	defaultTestTimeout = 10 * time.Second
+	defaultTestTimeout = 10 * time.Hour
```

Comment:
This needs to be reverted before merging.

Reply:
Whoops!